code/game/objects/items/tools/tool_behaviour.dm

```diff
@@ -16,7 +16,7 @@
 		var/datum/callback/tool_check = CALLBACK(src, .proc/tool_check_callback, user, target, amount, extra_checks)
 
 		if(ismob(target))
-			if(!do_mob(user, target, delay, extra_checks = tool_check))
+			if(!do_mob(user, target, delay, extra_checks = list(tool_check)))
```

Comment:
For some extra clarification, this is the only `do_mob` in the entire codebase which uses an `extra_checks`, so making do_mob take a list and putting this as a list here should be fine.